spec/acceptance/deb_admin_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'spec_helper_acceptance'
+
+describe 'Scenario: pulp-admin-deb' do
```

Comment:
This is not only the admin but actually the full deb functionality. That's great but the name should reflect it :)

Reply:
OK